docs/workstation_release_management.rst

```diff
@@ -14,176 +11,99 @@ SecureDrop Workstation code spans across several repositories:
+    ```shell
+    sha256sum bulid/debbuild/packaging/securedrop-foobar_x.y.z-rcN.deb
+    ```
+6. Save and publish your terminal history to the [build-logs repository](https://github.com/freedomofpress/build-logs/).
+
+7. Open a PR to https://github.com/freedomofpress/securedrop-dev-packages-lfs with the package you want to deploy. Remember to link to your build logs commit. Once your PR is merged, the package will be deployed to https://apt-test.freedom.press.
+
+## Step 3: Begin QA.
+
+You can start the QA process on the rc package that you deployed to https://apt-test.freedom.press in the previous step. If a bug is found, the fix can be added to the release branch and you can begin the rc process all over again with a new rc version.
+
+Once an rc package has been approved, you are ready to move on to the next step.
+
+## Step 4: Create a release tag.
+
+Begin this step on the day you want to release the package. It's best to start this process early in the day to ensure there is enough time for final QA checks, signing ceremonies, and stakeholder communications.
+
+1. Push a release tag on the same commit of the rc tag that was approved during QA.
+2. :ref:`Sign the tag with the SecureDrop release key` (or ask another maintainer to do this).
+
+## Step 5: Build a production version of the package.
+
+1. Once again, open a terminal in ``sd-dev-dvm``, clone ``securedrop-builder`` and install its dependencies.
+2. Build the package.
+3. Ouput the package hash so you can copy it into the build logs in a following step.
+    ```shell
+    sha256sum bulid/debbuild/packaging/securedrop-foobar_x.y.z.deb
+    ```
+4. Confirm the hash matches the x.y.z-rcN package that was approved for release.
+5. Save and publish your terminal history to the [build-logs repository](https://github.com/freedomofpress/build-logs/).
+
+## Step 6: Sign the Debian Release file with the SecureDrop release key.
+
+1. Add your package to a new branch called ``release`` in https://github.com/freedomofpress/securedrop-debian-packages-lfs.
+2. :ref:`Regenerate and sign the apt release file`.
+
+## Step 7: Deploy the package to ``apt-qa``.
+
+1. Push your ``release`` branch to remote to automatically deploy the package to https://apt-qa.freedom.press.
+2. Open a PR to merge ``release`` into ``main`` and link to the new `build-logs` commit.
+
+## Step 8: apt-qa preflight check
+
+Once the package has been deployed to https://apt-qa.freedom.press, ensure you are able to update and install it directly in the package's template VM by updating the apt sources file.
+
+## Step 9: Deploy the package to ``apt`` (prod)
+
+1. Merged the ``release`` branch into ``main`` to deploy your package to https://apt.freedom.press.
+2. Now that your package is on https://apt.freedom.press, you can run the updater to install your package in a production environment and ensure that it works as expected.
+
+Release an RPM package
+======================
+
+Release ``securedrop-workstation-dom0-config``
```

Comment:
Attaching the comment here, but it belongs a few lines below in text that hasn't changed (yet?). I noticed that there is no mention of conventional branch names in the RPM process, while the branch that will deploy the packages is conventionally named `release` in the Debian process. I don't know if that's expected, but I thought I'd mention it just in case.

Reply:
Yes, good observation. We don't have the same `release` branch mechanics for RPMs (yet? this might be a question to ask @legoktm in case he has plans of adding yum-qa.freedom.press like we have for apt-qa.freedom.press). For now, the docs are correct in not mentioning branch names because the names of branches don't matter for RPM releases.